### DISCUSSION TITLE
How to create a Zapier integration?

### DISCUSSION THREAD

**Discussion by thomasmol** [nextauthjs/next-auth#7714] [Help]
### Question 💬

I am trying to make my app work with Zapier and want to create an integration.

I am stuck at the authentication part, and need some help. I can't find any tutorials how to do this so I am asking here :)

I use the email provider and database session strategy so I don't have a username + password to setup the auth in Zapier.
Does anybody have some ideas how to do this?

### How to reproduce ☕️

Create a @auth/core app with the email provider and try to setup Zapier integration.

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by nucks** [comment]
@thomasmol did you ever figure this out?

**Reply by thomasmol** [reply]
Unfortunately, I wasn’t able to find a solution to make a Zapier integration. I think you need to setup your own OAuth server for this to work, which is not something nextauth/auth.js offers. In the end I built my own auth solution specifically for Zapier, which uses the simple api keys auth strategy.
